src/Components/Body/Body.js

```diff
@@ -0,0 +1,52 @@
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import BodyHeader from "./BodyHeader";
+import BodyAbout from "./BodyAbout";
+// import { Jumbotron, Container} from "reactstrap;"
+
+const Body = () => {
+  const initialState = [];
+  const [nasaData, setNasaData] = useState(initialState);
```

Comment:
nice work. Note you could also write lines 8 and 9 in one line:
const [nasaData, setNasaData] = useState([]);